script.js

```diff
@@ -1,4 +1,811 @@
+    (pcScore === WINNING_SCORE && pcCards.length === 2) ||
+    didPlayerStand
+  ) {
+    // Update betting points balance left at the end of each round
+    availableBettingPoints = calculateBettingPoints(
+      availableBettingPoints,
+      playerBet,
+      playerCards,
+      pcCards,
+      playerScore,
+      pcScore,
+      WINNING_SCORE
+    );
+
+    // Update high score
+    highestPointsRecord = updateHighestPointsRecord(availableBettingPoints);
+
+    // Turn Play button on while the other buttons off
+    playButton.disabled = false;
+    hitButton.disabled = true;
+    standButton.disabled = true;
+
+    // Reassign the main standard output message with all pc's cards info etc. when the round ends
+    myOutputValue = `You bet ${playerBet} points. <br> <br> You have:<br> ${allPlayerCardsInfo} with sum of <em>${playerScore}</em>.<br> <br> Computer has:<br> ${allPcCardsInfo} with sum of <em>${pcScore}</em>. <br> <br> ${resultMessage} <br> You have ${availableBettingPoints} points now. <br> <br> 🎆 HIGHEST RECORD: ${highestPointsRecord} points 🎆`;
+
+    // When betting points are running out, reset the betting points to 100
+    /* Reset as long as the final betting points/ slider max value <= 1 as:
+    (1) there may be cases of betting points balance being a decimal number < 1, where the game will then stuck due to the minimum slider value being set to 1 initially and can no longer be dragged
+    (2) the slider may be stucked too when the slider min value is 1 while its max value is also set to 1 but player can no longer slide it to change the value
+    */
+    if (availableBettingPoints <= 1) {
+      myOutputValue = `You bet ${playerBet} points. <br> <br> You have:<br> ${allPlayerCardsInfo} with sum of <em>${playerScore}</em>.<br> <br> Computer has:<br> ${allPcCardsInfo} with sum of <em>${pcScore}</em>. <br> <br> ${resultMessage} <br> You have ${availableBettingPoints} points now. <br> Game will restart from 100 points. <br> <br> 🎆 HIGHEST RECORD: ${highestPointsRecord} points 🎆`;
+      availableBettingPoints = 100;
+    }
+
+    // Reset the game in the end while having the values of available betting points and highest record persisted across multiple rounds
+    reset();
+  }
+
+  // --- If no one has blackjack and player has not chosen to stand, update the initial standard output message which shows only pc's 1st card ---
+
+  // If player has busted, prompt player to stand
+  else if (playerScore > WINNING_SCORE) {
+    myOutputValue += `You have busted! Please press Stand to see the result.`;
+  }
+  // If player has reached 21 points, prompt player to stand
+  else if (playerScore === WINNING_SCORE) {
+    myOutputValue += `You cards sum up to ${WINNING_SCORE}! Please press Stand to see the result.`;
+  }
+  // Else, in all other cases, prompt player to hit or stand
+  else {
+    myOutputValue += `Please press Hit to draw more card or press Stand to end your turn.`;
+  }
   return myOutputValue;
 };
+
+/*
```

Comment:
can move this whole commented out block into a different file for testing in the future, not necessary for fundamentals, but just take note next time